python/programme/check-prime-number/solution.py

```diff
@@ -0,0 +1,8 @@
+def check_prime(num):
+    for i in range(2, num):
```

Comment:
For improvement, check only for range (`2`, `num/2`)

### Explanation

For n = 15
- If n is divisible by any number in range(2, 7), it will not be a prime number.

For n = 31
- If n is divisible by any number in range(2, 15), it will not be a prime number.

### Benefit

We reduce no. of iteration and runtime

Reply:
Hi Harshraj, thanks for the feedback. I'll make that change as soon as possible.